hadoop-hdds/hadoop-dependency-server/pom.xml

```diff
@@ -71,6 +71,22 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
           <groupId>com.sun.jersey</groupId>
           <artifactId>jersey-json</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>log4j</groupId>
+          <artifactId>log4j</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>ch.qos.reload4j</groupId>
+          <artifactId>reload4j</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-log4j12</artifactId>
```

Comment:
i haven't checked but was it necessary to exclude slf4j-log4j12 and log4j?

Reply:
I don't think they are strictly necessary.  They used to be excluded in client dependencies, but not in server dependencies.  I added them here for consistency.